src/app/forms/utils/error-message-map.ts

```diff
@@ -1,3 +1,5 @@
 export const ErrorMessageMap: { [key: string]: any } = {
-  ['required']: (err: boolean, label: string) => `${label ? label : 'This filed'} is required`
+  ['required']: (err: boolean, label: string) => `${label ? label : 'This filed'} is required`,
+  ['pattern']: (err: boolean, label: string) => `${label ? label : 'This filed'} must match a pattern`,
```

Comment:
Are these supposed to be "This field"?

Reply:
Yes it's the default in case the label doesn't exist